terraform/main.tf

```diff
@@ -18,3 +18,11 @@ resource "aws_ecr_repository" "c14_team_growth_lmnh" {
 output "ecr_repository_id" {
   value = aws_ecr_repository.c14_team_growth_lmnh.id
 }
+
+resource "aws_s3_bucket" "gamma_long_term_storage" {
```

Comment:
Could you make the bucket name something more descriptive for our project? i.e. c14-team-growth-gamma-long-term-storage

Reply:
I feel like that is a bit long, could it be c14-team-growth-storage?

Reply:
Yes, even better!